www-client/uget/uget-2.2.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils xdg-utils
+
+IUSE="+gnutls aria2 gstreamer appindicator libnotify nls openssl sockets rss"
+KEYWORDS="~amd64 ~arm ~x86"
+SRC_URI="mirror://sourceforge/urlget/${P}.tar.gz"
+
+DESCRIPTION="Download manager using gtk+ and libcurl"
+HOMEPAGE="http://www.ugetdm.com"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+
+DOCS=( AUTHORS ChangeLog NEWS README )
+
+REQUIRED_USE="^^ ( gnutls openssl )"
+RDEPEND="
+	>=net-misc/curl-7.19.1
+	dev-libs/libpcre
+	>=dev-libs/glib-2.32:2
+	>=x11-libs/gtk+-3.4:3
+	gnutls? ( net-libs/gnutls dev-libs/libgcrypt:= )
+	openssl? ( dev-libs/openssl:0 )
+	gstreamer? ( media-libs/gstreamer:1.0 )
+	libnotify? ( x11-libs/libnotify )
+	aria2? ( net-misc/aria2[xmlrpc] )
+	appindicator? ( dev-libs/libappindicator:= )
+	"
+DEPEND="${RDEPEND}
+	dev-util/intltool
+	virtual/pkgconfig
+	sys-devel/gettext
+	"
+
+src_configure() {
+	econf $(use_enable nls) \
```

Comment:
For such a long list, I would suggest doing it like this (non-obligatory):

```
local myconf=(
    $(use_with ...)
    ...
)

econf "${myconf[@]}"
```

Then you don't need to play with backslashes.